### PR TITLE
Feature/i18n enhancement

### PR DESCRIPTION
**Pull request by ncoe**
Adding some unittests to the translation framework in the vibe.web.web module.
Adding support for the msgctxt tag that can be specified in po files.

### CODE REVIEW

File: source/vibe/web/i18n.d [hunk 1]
```diff
@@ -191,27 +193,67 @@ package template GetTranslationContext(alias METHOD)
...
+ * msgctxt - disabmbiguating context, like variable scope (optional, defaults to null)
+ * msgid - key to translate from (required)
+ * msgid_plural - plural form of the msg id (optional)
+ * msgstr - value to translate to (required)
+ * msgstr[0] - indexed translation for handling the various plural forms
+ * msgstr[1] - ditto
+ * msgstr[2] - ditto and etc...
+ */
+
+// TODO: Handle msgctxt
```
Comment (s-ludwig) [inline source/vibe/web/i18n.d]: I there something still left for msgctxt?

**Comment by s-ludwig** [comment]
I'd pull this and then remove the commented out code if there are no objections.

File: source/vibe/web/i18n.d (hunk 1 above)
Comment (ncoe) [inline source/vibe/web/i18n.d]: The msgcntxt should be complete, and that TODO can be removed. The only thing remaining should be additional overloads that support plurals and context for plurals.

**Comment by s-ludwig** [comment]
Okay, thanks! I've removed the commented out code; if necessary, we can always dig it up from the GIT history.

**Pull request merged**